TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
Without this change I am seeing the below error when restoring this project

>Errors in /Users/sridhar/github/cli/TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json
    System.Security.Cryptography.Algorithms 4.2.0-rc3-24113-00 provides a compile-time reference assembly for System.Security.Cryptography.Algorithms on .NETStandard,Version=v1.5, but there is no run-time assembly compatible with osx.10.11-x64.
    One or more packages are incompatible with .NETStandard,Version=v1.5 (osx.10.11-x64).

`NETStandard.Library` package is pulling down `System.Security.Cryptography.Algorithms`